Integrations/src/main/java/io/deephaven/integrations/python/PythonMergedListenerAdapter.java

```diff
@@ -91,6 +102,20 @@ public ArrayList<TableUpdate> currentRowsAsUpdates() {
 
     @Override
     protected void process() {
-        pyCallable.call("__call__");
+        pyListenerCallable.call("__call__");
+    }
+
+    @Override
+    protected void propagateErrorDownstream(boolean fromProcess, @NotNull Throwable error,
+            TableListener.@Nullable Entry entry) {
+        if (!pyOnFailureCallback.isNone()) {
```

Comment:
Should this log an error if `isNone() == True`?  Does this need to call a super method like the other Java listener does?